client/js/controllers/editor.js

```diff
@@ -831,6 +831,64 @@
                     }
                 };
 
+             $scope.importFile = function () {
+                    var file = filePath;
+                    if (file) {
+                        if (isLeaf) {
+                            var cached = file.split("/");
+                            cached.pop();
+                            file = cached.join("/");
+                        }
+                        var dialog = $('<div></div>').
+                                   html($compile('<form ng-controller="editor"> <input' +
+                                                 ' type="file" class="inputControls"' +
+                                                 ' multiple onChange="angular.element(this).scope().filesChanged(this)"/> <input type="button" class="inputControls" ng-click="uploadFiles()" value="Upload" /><li ng-repeat="file in files">{{file.name}}</li></form>')($scope)).
+                        dialog({
+                          title: "Choose file to import",
+                          autoOpen: false,
+                          modal: true,
+                          position: { at: "center top"},
+                          height: 167,
+                          width: 300,
```

Comment:
height and width too small to me here,

I put the following values:
height: 225
width: 300